README.md

```diff
@@ -189,12 +195,12 @@ You can also use `kubectl-ai` as a MCP server that exposes `kubectl` as one of t
 
 kubectl-ai project includes [k8s-bench](./k8s-bench/README.md) - a benchmark to evaluate performance of different LLM models on kubernetes related tasks. Here is a summary from our last run:
 
-| Model | Success | Fail |
-|-------|---------|------|
-| gemini-2.5-flash-preview-04-17 | 10 | 0 |
-| gemini-2.5-pro-preview-03-25 | 10 | 0 |
-| gemma-3-27b-it | 8 | 2 |
-| **Total** | 28 | 2 |
+| Model                          | Success | Fail |
+| ------------------------------ | ------- | ---- |
+| gemini-2.5-flash-preview-04-17 | 10      | 0    |
+| gemini-2.5-pro-preview-03-25   | 10      | 0    |
+| gemma-3-27b-it                 | 8       | 2    |
+| **Total**                      | 28      | 2    |
```

Comment:
Would it be possible to separate non-translation related changes in a separate PR ? The reason is because translation related changes merge will depend on how much of that can be automated but these can go before that.

Reply:
This part was automatically converted by the VSCode editor. I'm not sure which Extension completed it, but it automatically formatted the README to make it more organized.